### PR TITLE
Remove hardcoded twitter and kafka params.

### PR DESCRIPTION
**Pull request by pchanne**
Move configs to configuration/app.conf. Pass this file with '--properties-file /configuration/app.conf' with spark-submit

### CODE REVIEW

**Comment by jimbobhickville** [comment]
One minor suggestion, name the folder just ```conf``` to be consistent with most unix-y things.

**Comment by jimbobhickville** [comment]
Also, does this override the default spark-defaults.conf?  We probably want both of them.

File: examples/twitter_sentiment_v2/backend/configuration/app.conf [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#Twitter Credentials
+spark.sentimentApp.apiKey=
+spark.sentimentApp.apiSecret=
+spark.sentimentApp.accessToken=
+spark.sentimentApp.accessTokenSecret= 
+
+#Kafka config
+spark.sentimentApp.twitterRefresh=
```
Comment (jimbobhickville) [inline examples/twitter_sentiment_v2/backend/configuration/app.conf]: Can we put default values in this section?  I don't think customers will change these much, if at all.

**Comment by pchanne** [comment]
Okay I will update it to conf. 
Also, those parameters are in addition to spark-defaults.conf.

File: examples/twitter_sentiment_v2/backend/configuration/app.conf (hunk 1 above)
Comment (pchanne) [inline examples/twitter_sentiment_v2/backend/configuration/app.conf]: Yeah I can do that.

Comment (pchanne) [inline examples/twitter_sentiment_v2/backend/configuration/app.conf]: What values should I default?

Comment (jimbobhickville) [inline examples/twitter_sentiment_v2/backend/configuration/app.conf]: The ones that used to be in the app itself.

```
val twitterRefresh = 10
val kafkaBrokers = "slave-1.local:6667,slave-2.local:6667,slave-3.local:6667"
val kakfaQueue = "scored-tweets"
val numKafkaPartitions = 3
```

Comment (pchanne) [inline examples/twitter_sentiment_v2/backend/configuration/app.conf]: Cool. I will update it.

**Comment by jimbobhickville** [comment]
+1

**Pull request merged**